pkg/remote/resource_enumeration_error_handler.go

```diff
@@ -76,10 +132,18 @@ func handleAWSError(alerter alerter.AlerterInterface, listError *remoteerror.Res
 	return reqerr
 }
 
-func sendEnumerationAlert(provider string, alerter alerter.AlerterInterface, listError *remoteerror.ResourceEnumerationError) {
+func sendEnumerationAlert(provider string, alerter alerter.AlerterInterface, listError *remoteerror.ResourceScanningError) {
```

Comment:
Why do we have two distinct func `sendEnumerationAlert` and `sendDetailsFetchingAlert` ? 

The only thing that changes is `EnumerationPhase` or `DetailsFetchingPhase` ?

Maybe we can have a single `func sendAccessDeniedAlert` with the phase as parameter ?

Reply:
personally I like that there is two functions for readability

Reply:
I created a common sendAlert() function that maintain readability